### PR TITLE
LaTeX syntax highlighting in Rmd

### PR DESCRIPTION
**Pull request by storopoli**
**What problem did you solve?**

LaTeX syntax highlighting in Rmd files #93 

**(If you have)Screenshot**

**(If you do not have screenshot) How can I check this pull request?**
You need to test it. Unfortunately I don't know how to test it.

### CODE REVIEW

File: syntax/Markdown Redcarpet.json [hunk 1]
```diff
@@ -137,10 +138,28 @@
...
+                    "end" : "([$|$$])",
+                    "endCaptures" : {
+                            "1" : {
+                            "name" : "punctuation.definition.raw.rmarkdown"
+                        }
+                    },
+                    "contentName" : "meta.embedded.block.latex",
+                    "patterns" : [
+                        {"include" : "source.latex"}
+                    ]
```
Comment (andycraig) [inline syntax/Markdown Redcarpet.json:158]: There is a missing `}` after this line.

File: syntax/Markdown Redcarpet.json [hunk 2]
```diff
@@ -137,10 +138,28 @@
...
             }
         }
     },
     "scopeName" : "text.html.markdown.redcarpet",
     "uuid" : "BE79A69A-B9F5-4BCD-9068-893496E86663"
+    }
```
Comment (andycraig) [inline syntax/Markdown Redcarpet.json:164]: This `}` shouldn't be here.

**Comment by Ikuyadeu** [comment]
LGTM

**Review by Ikuyadeu — APPROVED** [review]

**Comment by andycraig** [comment]
@Ikuyadeu I think this PR is still in progress. I don't think it provides LaTeX highlighting yet.

**Comment by Ikuyadeu** [comment]
@andycraig OK, I make it to draft

**Comment by boyesjo** [comment]
Any updates?

**Comment by storopoli** [comment]
No

**Comment by xiajunwen1007** [comment]
Now vscode has a built-in support for latex syntax. Maybe this would work now.

**Comment by eitsupi** [comment]
It didn't work as is, but I changed it as shown in the image below and tested it, and the latex highlighting came on. However, there is something wrong.

image

image

I imagine that the inclusion of dollar signs in the latex syntax complicates things.

**Pull request closed without merging**